test/logs.js

```diff
@@ -107,6 +107,7 @@ QUnit.test( module1Test1.name, function( assert ) {
 	);
 
 	delete logContext.runtime;
+	delete logContext.source;
```

Comment:
this is a harming flag to the tests, we still need to test that tests contains this source property, even if we than later remove it here to test exact values on other tests.